### PR TITLE
removed buy button from NFT card and fixed View page to be formatted …

### PR DESCRIPTION
**Pull request by krystyna-urbanczyk**
…properly.

### CODE REVIEW

File: kraken-app/src/Components/CardViewPage/CardViewPage.jsx [hunk 1]
```diff
@@ -25,7 +26,7 @@ function CardViewPage(props) {
             </Grid>
             <Grid item xs={5}>
                 <Typography variant="h4" component="div" gutterBottom>
-
```
Comment (nathanclough) [inline kraken-app/src/Components/CardViewPage/CardViewPage.jsx:28]: We only have one name right now so this section can be removed

**Review by nathanclough — CHANGES_REQUESTED** [review]
The section with "name?" should be removed at least for demo we can update metadata later to have more fields

File: kraken-app/src/Components/CardViewPage/CardViewPage.jsx (hunk 1 above)
Comment (krystyna-urbanczyk) [reply kraken-app/src/Components/CardViewPage/CardViewPage.jsx:28]: Shouldn't it say the name then?

Comment (krystyna-urbanczyk) [reply kraken-app/src/Components/CardViewPage/CardViewPage.jsx:28]: What is the one name?

**Review by nathanclough — APPROVED** [review]

**Pull request merged**